addons/website/static/src/js/post_link.js

```diff
@@ -9,6 +9,22 @@ publicWidget.registry.postLink = publicWidget.Widget.extend({
     events: {
         'click': '_onClickPost',
     },
+
+    /**
+     * Allows the link to be interacted with only when Javascript is loaded.
+     *
+     * @override
+     */
+    start() {
+        const def = this._super(...arguments);
+        this.el.classList.add('o_post_link_js_loaded');
```

Comment:
Technically, when I see a class addition in a public widget... I expect that class to be removed in the destroyed. Only really important if any "post_link" can actually be saved after edit mode though... but I would still add it for consistency / stable safety